### PR TITLE
Add OR operator support.

### PR DESCRIPTION
**Pull request by timesong**
Make sql include OR operator, for example:

`print select('orders', {'status': 1,
    '|': [
        {'code1': '00001'},
        {'code2': '00002'}
    ]
})`

will output:

`SELECT * FROM "orders" WHERE "status" = 1 AND (("code1" = '00001') OR ("code2" = '00002'))`

### CODE REVIEW

**Comment by uranusjr** [comment]
The syntax doesn’t make sense to me. If it is up to me, I would make a `mosql.op.or_` class like this

```pycon
>>> from mosql.op import or_
>>> print(select('orders', {'status': 1, or_({'code1': '00001', 'code2': '00002'})}))
SELECT * FROM "orders" WHERE "status" = 1 AND (("code1" = '00001') OR ("code2" = '00002'))
>>> print(select('orders', {'status': 1, or_([('code1', '00001'), ('code2', '00002')])}))
SELECT * FROM "orders" WHERE "status" = 1 AND (("code1" = '00001') OR ("code2" = '00002'))
```

Also, `|` means bitwise or is both SQL and Python, so even if you opt for this syntax, you need to choose another key.

File: tags [hunk 1]
```diff
@@ -0,0 +1,588 @@
+!_TAG_FILE_FORMAT	2	/extended format; --format=1 will not append ;" to lines/
```
Comment (moskytw) [inline tags]: You committed the tags file?

**Comment by moskytw** [comment]
@timesong Thanks. It's a good idea. I actually want to add it a long time ago. But I believe we can have a better implementation, I will add it in the next version.

**Comment by moskytw** [comment]
btw I also agree with @uranusjr. I would prefer using “OR” as the key.

**Pull request closed without merging**